packages/flat-components/src/components/FlatIcons/icons/SVGMaximize.tsx

```diff
@@ -0,0 +1,31 @@
+import "../style.less";
+import React from "react";
+import { FlatIconProps } from "../types";
+
+export const SVGMaximize: React.FC<FlatIconProps> = ({ active, className = "", ...restProps }) => {
+    return (
+        <svg
+            className={`${className} flat-icon ${active ? "is-active" : ""}`}
```

Comment:
I could be `classNames(className, "flat-icon", active && "is-active")`.

Reply:
nice advice ! !  before write is script generated, so that keep it consistent. the next pr will take your advice.

Reply:
I am sorry, after our discussion we don't want to include the classNames package here 😔

Reply:
It's OK. That's up to you.

Reply:
By the way, here is a lighter alternative package of `classnames`.

FYI: https://github.com/sindresorhus/class-names